app/Models/Movie.php

```diff
@@ -22,6 +22,7 @@ public function genres() {
     }
 
     public function celebs() {
-        return $this->belongsToMany(Celeb::class, 'celeb_movie');
+        return $this->belongsToMany(Celeb::class, 'celeb_movie')
+            ->withPivot(['character_name']);
```

Comment:
You do not need the array here Jayd 😄 , just `withPivot('character_name')` should work